### ISSUE TITLE
(low) [Listing page] What is the external link icon for?

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-course-list#112]
The main reason I ask is because there is no alternative text and it has the aria-hidden attribute. If it is communicating something significant should it have alternative text? If it is not communicating anything significant, should it be there at all?

**Comment by leticiaseixas** [comment]
The alternative text has been included, and the aria-hidden removed.

But the question whether the icon should be there at all is relevant.

This icon is used to indicate external links on some of the pages on the WAI website (e.g. [https://www.w3.org/WAI/planning/statements/](https://www.w3.org/WAI/planning/statements/)), but for some pages this icon is not used (e.g. [https://www.w3.org/WAI/](https://www.w3.org/WAI/)).

Maybe the new resources are no longer using it? What pattern should we follow?

@shawna-slh Any thoughts on this?

**Comment by shawna-slh** [comment]
Hi @leticiaseixas We use the icons with links or not depending on context. Basically, will users be surprised and annoyed if they click on a link and it goes to a different website? If so, then we put an icon with the link to inform them before they click on it.

For the course list (and the tools lists @SteveALee ), I think users won't be particularly surprised with they click:
> [Course or Tool Name](@@) by Provider/Company

Therefore, I think we should not have icons after the Course or Tool Name. Icons are unnecessary clutter in this context.

Thanks for checking!
